mdm/orientdb-api/src/main/java/com/gentics/mesh/graphdb/MeshOrientGraphEdgeQuery.java

```diff
@@ -54,6 +54,7 @@ public Iterable<Edge> fetch(Optional<? extends Collection<? extends Class<?>>> m
 		text.append(QUERY_FROM);
 		text.append(OrientBaseGraph.encodeClassName(edgeLabel));
 		text.append(" ");
+		buildOrderEntityRequest(text, true, fetchPlan == null);
```

Comment:
Long story short: OrientDB SQL is cryptic in a manner that there are a plenty of ways to request derived data with no sign of a syntax error, but not all of them lead to the expected result. Here a `inV()[edgefilter].outV()` form has been rewritten to a `LET $param = ...` form, because the former did not produce the sorted results for some cases. One can set the `<logger name="com.gentics.mesh.graphdb" level="DEBUG"/>` up to observe the actual queries.